dist/cli/list.js

```diff
@@ -13,14 +13,14 @@ exports.list = list;
 
 var _fmt = require("../fmt");
 
+var _fmt2 = _interopRequireDefault(_fmt);
```

Comment:
looks like newer babel version than in your current dist, this can be ignored with my another PR